guides/common/modules/proc_building-a-discovery-image.adoc

```diff
@@ -7,7 +7,11 @@ Discovered hosts keep running the Discovery image until they are rebooted into A
 ifdef::satellite[]
 The operating system image is based on {RHEL} 7.
 endif::[]
-ifndef::satellite[]
+ifdef::orcharhino[]
+The operating system image is based on Rocky Linux 8.
+
+The Discovery image is based on Cent OS 8.
```

Comment:
CentOS 8 is EoL.

Reply:
It is based on CentOS 8 Stream really, we build against a rolling distro.

Reply:
@lzap Note that this is specifically for the orcharhino build. I don't know what they build on. Upstream is just below in the `ifndef`.

Reply:
Thanks for clarifing. Then this needs to be changed to "CentOS 8 Stream". @jloeser